e2e/cypress/integration/channel/message_spec.js

```diff
@@ -140,4 +141,63 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('MM-16730 Reply to an older message', () => {
+        cy.getCurrentChannelId().then((channelId) => {
+            // # Get yesterdays date in UTC
+            const yesterdaysDate = new Date().getTime() - 86400000;
```

Comment:
Nit pick, but could leverage moment (included in Cypress) for possibly easier "historic" dates.

Reply:
Hooray for moment 🎉 Updated in e84aae0.